ZooManagement/ZooManagement_1.sql

```diff
@@ -0,0 +1,118 @@
+CREATE DATABASE ZooManagement;
```

Comment:
I love the fact that you also created the db structure and played around with SQL queries. For sure it helped you practice SQL!